lib/cli/commands/connect.ts

```diff
@@ -11,10 +11,16 @@ export const builder = {
     description: 'uri of peer in [nodePubKey]@[host]:[port]',
     type: 'string',
   },
+  retry_connecting: {
+    description: 'whether to apply multiple connection retry attempts',
```

Comment:
I'm not that sure this is a useful feature, wouldn't rpc call potentially take a very long time?

Reply:
Yes, it will be blocked until connection will succeed. 
Our options is either to propagate updates (same ones from the logs), or to remove this feature.

Reply:
Gotcha. I'd lean towards removing this, maybe we can discuss in gitter and see what others think.

Reply:
I removed this feature from cli/grpc.